### ISSUE TITLE
Locker wrapping pseudo-duplication

### ISSUE THREAD

**Issue by NigglyWiggly** [d3athrow/vgstation13#10771]
Using package wrap while pushing a locker at the correct time can create a sort of duplicate of it.
You will have a wrapped locker and a normal locker.
You can unwrap the locker to teleport the normal one to that location.
Placing objects inside the normal locker and then unwrapping the wrapped one will teleport those objects as well. 

To Reproduce, just push a locker while applying package wrap to it. The timing looks pretty specific.

Server Revision is: c76eaa7a828f3109e481c3e3430dfe4593f179f9

This issue was brought to my attention by Kranenberg.

**Comment by Shadowmech88** [comment]
Can you interact with the unwrapped locker without disturbing the wrapped one?

**Comment by NigglyWiggly** [comment]
Yep.
I watched Kranenberg make three copies of the same locker like this, and then put a toolbox in the unwrapped one, which then appeared inside the wrapped one when he opened it

**Comment by clusterfack** [comment]
Yeah the locker should be ending up inside the wrapped one no matter what you do, that's pretty bizarre. I feel the answer will be fairly obvious when looking at the wrapper code for it though.

**Comment by ComicIronic** [comment]
There is some problem with do_after, I've noticed that during lag you can pull off multiple construction steps at once with the right timing - it seems to depend on hitting the completion tick, possibly.

**Comment by clusterfack** [comment]
That's probably a problem with that specific construction style. I've never seen do_after fuck up ever, it's like the most consistent thing in the codebase.

**Comment by clusterfack** [comment]
So I looked into this and... I have no idea why what you described would occur.

**Comment by clusterfack** [comment]
Possibly as a result of mob/bump calling spawn and sleep a few times. Not 100 percent sure why they do thag

**Comment by Shadowmech88** [comment]
Can confirm this still exists. However, when I unwrapped the duplicate locker in local testing, nothing happened. The wrapped locker simply disappeared while the original remained in place.
Still, I'll leave the report open since even the creation of the empty wrapping at all is unintended.

**Comment by Exxion** [comment]
I know what fixed the teleportation but it's not important enough to go looking
As for the other part, likely what Cluster said, and also the same issue as #17012
